windows/cross-compile/build.sh

```diff
@@ -92,1399 +76,45 @@ then
-fi
-
-
-# Opus
-
-OPUS_PREFIX_DIR="$DEP_DIR/libopus"
-OPUS_VERSION=1.3.1
-# https://archive.mozilla.org/pub/opus/SHA256SUMS.txt
-OPUS_HASH="65b58e1e25b2a114157014736a3d9dfeaad8d41be1c8179866f144a2fb44ff9d"
-OPUS_FILENAME="opus-$OPUS_VERSION.tar.gz"
-if [ ! -f "$OPUS_PREFIX_DIR/done" ]
-then
-  rm -rf "$OPUS_PREFIX_DIR"
-  mkdir -p "$OPUS_PREFIX_DIR"
-
-  curl $CURL_OPTIONS -O "https://archive.mozilla.org/pub/opus/$OPUS_FILENAME"
-  check_sha256 "$OPUS_HASH" "$OPUS_FILENAME"
-  bsdtar --no-same-owner --no-same-permissions -xf "$OPUS_FILENAME"
-  rm $OPUS_FILENAME
-  cd opus*
-
-  LDFLAGS="-fstack-protector" \
-  CFLAGS="-O2 -g0" ./configure --host="$ARCH-w64-mingw32" \
-                               --prefix="$OPUS_PREFIX_DIR" \
-                               --enable-shared \
-                               --disable-static \
-                               --disable-extra-programs \
-                               --disable-doc
-  make
-  make install
-  echo -n $OPUS_VERSION > $OPUS_PREFIX_DIR/done
-
-  cd ..
-  rm -rf ./opus*
-else
-  echo "Using cached build of Opus `cat $OPUS_PREFIX_DIR/done`"
-fi
-
-
-# Sodium
-
-SODIUM_PREFIX_DIR="$DEP_DIR/libsodium"
-SODIUM_VERSION=1.0.18
-SODIUM_HASH="6f504490b342a4f8a4c4a02fc9b866cbef8622d5df4e5452b46be121e46636c1"
-SODIUM_FILENAME="libsodium-$SODIUM_VERSION.tar.gz"
-if [ ! -f "$SODIUM_PREFIX_DIR/done" ]
-then
-  rm -rf "$SODIUM_PREFIX_DIR"
-  mkdir -p "$SODIUM_PREFIX_DIR"
-
-  curl $CURL_OPTIONS -O "https://download.libsodium.org/libsodium/releases/$SODIUM_FILENAME"
-  check_sha256 "$SODIUM_HASH" "$SODIUM_FILENAME"
-  bsdtar --no-same-owner --no-same-permissions -xf "$SODIUM_FILENAME"
-  rm "$SODIUM_FILENAME"
-  cd libsodium*
-
-  LDFLAGS="-fstack-protector" \
-  ./configure --host="$ARCH-w64-mingw32" \
-              --prefix="$SODIUM_PREFIX_DIR" \
-              --enable-shared \
-              --disable-static
-  make
-  make install
-  echo -n $SODIUM_VERSION > $SODIUM_PREFIX_DIR/done
-
-  cd ..
-  rm -rf ./libsodium*
-else
-  echo "Using cached build of Sodium `cat $SODIUM_PREFIX_DIR/done`"
-fi
-
-
-# VPX
-
-VPX_PREFIX_DIR="$DEP_DIR/libvpx"
-VPX_VERSION=v1.11.0
-VPX_HASH="965e51c91ad9851e2337aebcc0f517440c637c506f3a03948062e3d5ea129a83"
-VPX_FILENAME="libvpx-$VPX_VERSION.tar.gz"
-if [ ! -f "$VPX_PREFIX_DIR/done" ]
-then
-  rm -rf "$VPX_PREFIX_DIR"
-  mkdir -p "$VPX_PREFIX_DIR"
-
-  curl $CURL_OPTIONS https://github.com/webmproject/libvpx/archive/$VPX_VERSION.tar.gz -o $VPX_FILENAME
-  check_sha256 "$VPX_HASH" "$VPX_FILENAME"
-  bsdtar --no-same-owner --no-same-permissions -xf "$VPX_FILENAME"
-  rm $VPX_FILENAME
-  cd libvpx*
-
-  if [[ "$ARCH" == "x86_64" ]]
-  then
-    VPX_TARGET=x86_64-win64-gcc
-    # There is a bug in gcc that breaks avx512 on 64-bit Windows https://gcc.gnu.org/bugzilla/show_bug.cgi?id=54412
-    # VPX fails to build due to it.
-    # This is a workaround as suggested in https://stackoverflow.com/questions/43152633
```

Comment:
You didn't preserve these comments. They say that `-fno-asynchronous-unwind-tables` is there only because of that bug and it could be removed once the bug is fixed. If you remove the comment, no one will remove `-fno-asynchronous-unwind-tables` even when the bug is fixed.

Seriously, what's up with you removing useful comments?

Reply:
Mistake, fixed